src/main/java/com/socrata/datasync/HttpUtility.java

```diff
@@ -151,7 +151,7 @@ private boolean canUse(String option) {
 
         public boolean retryRequest(IOException exception, int executionCount, HttpContext context) {
             // Do not retry if over max retry count
-            if (executionCount >= 5)
+            if (executionCount >= 3)
```

Comment:
Well...  it looks like this might be tricker than I anticipated.

This method is called by Apache, not by our own client code.  Going to dig a little deeper.